CHANGES

```diff
@@ -185,6 +185,11 @@ coqdep
   This is used by coq_makefile when generating dependencies for .v
   files (but not other files).
 
+Infrastructure
+
+- The test suite allows writing unit tests against OCaml code in the Coq
+  code base. Those unit tests create a dependency on the OUnit test framework.
```

Comment:
Note that this change entry is not correctly located anymore (since the target is now 8.9). And anyways, this might be more suited for `dev/doc/changes.md` rather than `CHANGES` given it is not a relevant information for our users.

Reply:
How about this text in changes.md?  (Also, FWIW, it would be nice if the dev/doc info was better organized, maybe more as a unified document with a table of contents and an index.  That would also make it easier to see what things are not well covered, too.)

```
### Unit test support

  The test suite now allows writing unit tests against OCaml code in the Coq
  code base. Those unit tests create a dependency on the OUnit test framework.
  Tests are in test-suite/unit-tests.  Use `make unit-tests' in the unit-tests
  directory to run them. 
```

Reply:
For "is not correctly located anymore", better to say "is in the wrong place".  "correctly located" is odd by itself.

For "a relevant information": "information" is not countable so "a" would not be used.

Reply:
> How about this text in changes.md?

:+1: 

> FWIW, it would be nice if the dev/doc info was better organized, maybe more as a unified document with a table of contents and an index. That would also make it easier to see what things are not well covered, too.

I completely agree. This has been something that I wanted to do for a long time but I haven't gone down to it yet. Help welcome! Maybe we should start with a very simple README and make it grow bits by bits. Note also that much clean-up would be needed in this `dev/doc` repository. Cf. #7033 in particular.